src/main/java/org/cryptomator/common/keychain/KeychainModule.java

```diff
@@ -23,11 +23,12 @@ static List<KeychainAccessProvider> provideSupportedKeychainAccessProviders() {
 	@Singleton
 	static ObjectExpression<KeychainAccessProvider> provideKeychainAccessProvider(Settings settings, List<KeychainAccessProvider> providers) {
 		return Bindings.createObjectBinding(() -> {
+			if ( !settings.useKeychain().get() ) return null;
```

Comment:
please always use `{\n ... \n}`, even for simple expressions to avoid unnecessary diffs the next time we autoformat the code using Intellij IDEA.

If you use IDEA as well, just run "Reformat Code". The code style is committed to this repo